src/js/components/generateEF/GenerateEFPage.jsx

```diff
@@ -8,27 +8,48 @@ import Navbar from '../SharedComponents/navigation/NavigationComponent.jsx';
 import AddDataHeader from './../addData/AddDataHeader.jsx';
 import Progress from '../SharedComponents/ProgressComponent.jsx';
 
+import * as ReviewHelper from '../../helpers/reviewHelper.js';
+
 import GenerateEFContainer from '../../containers/generateEF/GenerateEFContainer.jsx';
 import GenerateEFError from './GenerateEFError.jsx';
 
+import GTASBanner from '../SharedComponents/GTASWarningBanner.jsx';
+
 export default class GenerateEFPage extends React.Component {
 
     constructor(props) {
         super(props);
 
         this.state = {
             showError: false,
-            errorMessage: ''
+            errorMessage: '',
+            gtas: null
         };
     }
 
+    componentDidMount() {
+        this.isGtas();
+    }
+
     showError(message) {
         this.setState({
             showError: true,
             errorMessage: message
         });
     }
 
+    isGtas() {
+        ReviewHelper.isGtasWindow()
+            .then((res) => {
+                if(res != this.state.gtas) {
+                    this.setState({gtas: res})
```

Comment:
We can still just set gtas to `res.data` here. Apply to all below. There is no reason not to because you're always passing "data" back, it is just either None or contains an object.